packages/react/src/react-chat.ts

```diff
@@ -109,15 +76,4 @@ export class Chat2<
       state: new ReactChatState(messages),
     });
   }
-
-  get status(): ChatStatus {
-    return this.state.getStatus();
-  }
-
-  get messages(): UIMessage<
-    MESSAGE_METADATA,
-    InferUIDataParts<UI_DATA_PART_SCHEMAS>
-  >[] {
-    return this.state.getMessages();
-  }
```

Comment:
No longer needed when `AbstractChat` already declares `status` and `messages` as getters